system-test/logging-winston.ts

```diff
@@ -226,6 +226,38 @@ describe('LoggingWinston', () => {
           errEvent.serviceContext.service, 'logging-winston-system-test');
       assert(errEvent.message.startsWith(`an error Error: ${message}`));
     });
+
+    it('reports errors when logging errors with winston3', async () => {
+      const start = Date.now();
+      const service = 'logging-winston-system-test-winston3';
+      const LoggingWinston = inject('../src/index', {
+                               winston: winston3,
+                               'winston/package.json': {version: '3.0.0'}
+                             }).LoggingWinston;
+
+      const logger = winston3.createLogger({
+        transports: [new LoggingWinston(
+            {logName: LOG_NAME, serviceContext: {service, version: 'none'}})],
+      });
+
+      const message = `an error at ${Date.now()}`;
+
+      // logger does not have index signature.
+      // tslint:disable-next-line:no-any
+      (logger as any)['error'](new Error(message));
```

Comment:
Could the cast and tslint-disable be avoided if this was written as `logger.error(new Error(message))`?

Reply:
Seems like it. No idea why its written like this =)